src/test/java/api/loans/scenarios/ChangeDueDateAPITests.java

```diff
@@ -377,10 +377,6 @@ void dueDateChangeShouldClearRenewalFlagWhenSetAndNoOpenRecallsInQueue() {
       .fulfilToHoldShelf(servicePointsFixture.cd1()));
 
     Response recalledLoan = loansClient.getById(initialLoan.getId());
-    ZonedDateTime recalledLoanDueDate = ZonedDateTime.parse(recalledLoan.getJson().getString("dueDate"));
-
-    assertThat(recalledLoan.getJson().getBoolean("dueDateChangedByRecall"), equalTo(true));
-    assertThat(calculateDaysBetween(initialDueDate, recalledLoanDueDate), equalTo(16));
```

Comment:
I think it makes sense to pull the fix from master branch directly to the target branch to avoid some confusion in the PR